src/audio/pipeline.c

```diff
@@ -26,6 +26,7 @@
 #include <errno.h>
 #include <stddef.h>
 #include <stdint.h>
+#include <sof\lib\memory_pm.h>
```

Comment:
Includes order goes from local to global, so it should be before stdint etc.

Reply:
it was for test of functionality will be removed in final pr